### DISCUSSION TITLE
When i push new changes from my local repo its not pushed by me

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When i push my new changes from my local repo and check the changes on github, it says that the changes where made by a different account  (not gonna say the name). Its not my account i never have seen this human beeing and i dont get why it doesnt show my account on the changes.  it shows this random account on anything i do on my local machine (creating dirs, files etc.). The repo is private and it shows my commit messages.

It seems like a bug to me that could be exploited. 
Would be nice if somebody could help me :)

**Comment by jge162** [comment]
Check `$ git config --list` make sure info is correct.

**Comment by jge162** [comment]
If it is wrong then 

`git config --global user.name "Your Name"`
`git config --global user.email "your.email@example.com"`

**Reply by ghost** [reply]
Thanks for the reply,
its configured correctly. The guy that is linked to the upload is a valid indian web developer its not just a wrong name, its a valid github user. The uploads are linked to a wrong account

**Comment by MrBam44** [comment]
If the user.name and user.email are not set correctly, you can update them by running the following commands:
`$ git config --global user.name "Your Name"`

`$ git config --global user.email "your.email@example.com"`
Make sure to replace "Your Name" and "[your.email@example.com](mailto:your.email@example.com)" with your own information. After running these commands, try committing changes again and see if your name and email appear correctly on GitHub.

**Comment by SoltanHuseynov** [comment]
step 1: you have to check account on the github. if this not solution
step 2:  you to change the email address in your local Git configuration on 'setting your commit email address'
step3: adding an email address to your GitHub account.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
